tests/www/test_views.py

```diff
@@ -523,9 +523,9 @@ def test_task_stats(self):
         self.assertEqual(resp.status_code, 200)
 
     def test_task_stats_only_noncompleted(self):
```

Comment:
How about we change this too?

```suggestion
    @conf_vars({("webserver", "show_recent_stats_for_completed_runs"): "False"})
    def test_task_stats_only_noncompleted(self):
```

Reply:
Done!